src/operator/activation-inl.h

```diff
@@ -24,7 +24,7 @@ namespace op {
 namespace activation {
 enum ActivationOpInputs {kData};
 enum ActivationOpOutputs {kOut};
-enum ActivationOpType {kReLU, kSigmoid, kTanh};
+enum ActivationOpType {kReLU, kSigmoid, kTanh, kSoftplus};
```

Comment:
These days it's more commonly known as SoftReLU I think.
Consider change the naming